static/templates/items/weapon-details.hbs

```diff
@@ -96,7 +96,7 @@
                     name="system.runes.property.{{index}}"
                     id="{{../fieldIdPrefix}}runes-property-{{index}}"
                     {{disabled slot.disabled}}
-                    {{#if @root.item.isSpecific}}class="readonly"{{/if}}
+                    class="{{#if @root.item.isSpecific}}readonly{{/if}}{{#if slot.adjusted}} adjusted{{/if}}"
```

Comment:
`readonly` isn't an attribute of select elements

Reply:
It is a CSS class tho.

Reply:
Ope